docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -13,11 +13,23 @@ description: Learn how to securely define and access environment variables in yo
 
 How environment variables are declared depends on how your Dagster project is deployed.
 
-### Dagster Cloud
+<TabGroup>
+  <TabItem name="Local development">
 
-Environment variables can be set a variety of ways in Dagster Cloud. Refer to the [Dagster Cloud environment variables guide](/dagster-cloud/developing-testing/environment-variables-and-secrets) for more info.
+If your Dagster project contains a `.env` file, Dagster will automatically load the environment variables it contains when Dagit is started.
```

Comment:
Should we give an example of what a .env file looks like here?

Reply:
Is there a benefit to that?

Reply:
I'm just not sure that it's a universally understood thing (the key=value format for example might not be obvious). I'd be happy to defer to whatever vercel does here though

Reply:
Oh, Vercel <3 I don't mind putting that in here to make it clear; I was just curious.